### ISSUE TITLE
Unresolved reference setBody in generated API implementations

### ISSUE THREAD

**Issue by padmalcom** [Foso/Ktorfit#726]
### Ktorfit version

2.1.0

### What happened and how can we reproduce this issue?

When building my multiplatform project, the implementation of the following API lacks the import 'import io.ktor.client.request.setBody'.

```
interface FileApi {
    @GET(Endpoints.FILE_BY_ID)
    suspend fun downloadFile(
        @Path("fileId") fileId: String
    ): Response<ByteArray>

    @Multipart
    @POST(Endpoints.FILES)
    @Headers("$HEADER_NO_BODY_LOGGING: true")
    suspend fun uploadFile(
        @Part file: PartData
    ): Response<Boolean>
}
```

### What did you expect to happen?

I'd expect ktorfit to add the missing import.

### Is there anything else we need to know about?

We are grateful for your effort with this nice library.

**Comment by Foso** [comment]
Hi @padmalcom , thank you for the bug report! I had not noticed the bug because i always had a some other function in the same file where i used @Body and that added the import

**Comment by padmalcom** [comment]
Hi @Foso, thank you for your fast reply. So for the time waiting I add a dummy interface adding a @Body annotation to get the required import.

**Comment by yurii-shcherbiuk** [comment]
@Foso ksp 1.0.27 
<img width="719" alt="Знімок екрана 2024-11-09 о 18 50 24" src="https://github.com/user-attachments/assets/d2f9bcb0-2a6c-4a32-b4a5-e915801ded29">

**Comment by Foso** [comment]
@yurii-shcherbiuk Thank you for the information! The imports are now fixed with Ktorfit 2.2.0
